pennylane/noise/conditionals.py

```diff
@@ -490,7 +490,7 @@ def __init__(self, mps):
             self._cmps.append(mp if isclass(mp) else mp.__class__)
 
         mp_ops = list(
-            op.__class__.__name__ for op in self.condition
+            getattr(op, "__name__", repr(op)) for op in self.condition
```

Comment:
```suggestion
            getattr(op, "__name__", op.__class__.__name__) for op in self.condition
```

The idea was to have the same label for both cases:
```py
>>> qml.noise.meas_eq(qml.expval), qml.noise.meas_eq(qml.expval(qml.X(0)))
(MeasEq('ExpectationMP'), MeasEq('expval(X(0))')) #  (MeasEq('ExpectationMP'), MeasEq('ExpectationMP')) 
```